pftp/proxy.go

```diff
@@ -244,7 +250,7 @@ func (s *proxyServer) switchOrigin(clientAddr string, originAddr string, tlsProt
 			s.log.debug("reconnect to origin")
 			cnt++
 
-			s.origin.Close()
```

Comment:
Could you tell me the meaning of the change?

Reply:
It's just same with s.origin.Close() but want to match the function used in client_handler.